src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-policy-definition-audit-effect/main.bicep

```diff
@@ -0,0 +1,21 @@
+﻿// $1 = policyDefinition
+// $2 = 'name'
+// $3 = 'displayName'
+// $4 = 'Custom'
+// $5 = 'All'
+// $6 = 'description'
+// $7 = '0.1.0'
+// $8 = 'category'
+// $9 = 'source'
+// $10 = parameterName
+// $11 = 'String'
+// $12 = 'defaultValue'
+// $13 = 'displayName'
+// $14 = 'description'
+// $15 = allOf
+// $16 = 'field'
+// $17 = equals
+// $18 = 'conditionValue'
+// $19 = 'Audit'
+
```

Comment:
Add `targetScope = 'subscription'` to this file here, and re-generate baselines.

Reply:
done!